src/freenet/clients/fcp/RequestStatus.java

```diff
@@ -63,10 +69,15 @@ synchronized void restart(boolean started) {
 		this.totalBlocks = total;
 		this.minBlocks = min;
 		this.fetchedBlocks = fetched;
+        // clone() because Date is mutable
+		this.latestSuccess
```

Comment:
That one's editor has bugs hardly seems like a sufficient excuse to write messy whitespace, and one cannot rule out that people set their editors to display whitespace explicitly. (Spoilers: I do!) If you don't want to fix this I can do it after we merge this.